### PR TITLE
feat(app): #1590 updated icons to use material icons

### LINKED ISSUE #1590: Remove icons from assets directory and import correctly
Import icons the standard way and remove the icon files from the assets directory

### PR DESCRIPTION
**Pull request by webkhushboo**
closes #1590 

### Notes

A summary of what was achieved in this PR

- [ ] Updated all icons to use Angular material icons

### CODE REVIEW

File: web/src/app/app.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <mat-sidenav-container class="sidenav-container">
   <mat-sidenav #sidenav [mode]="sideNavMode" class="sidenav" [ngClass]="{'sidenav__open': menuTriger}" fixedInViewport="false" role="navigation" [opened]="!isSmallScreen">
     <div class="heading" (click)="openMenu($event)">
-      <mat-icon svgIcon="logo_icon" class="heading__logo"></mat-icon>
+      <img src="assets/icons/Logo.svg" class="heading__logo"/>
```
Comment (eddiejaoude) [inline web/src/app/app.component.html]: `Logo.svg` should be lowercase `logo.svg`

File: web/src/app/app.component.html [hunk 2]
```diff
@@ -83,7 +83,7 @@
       </div>
       <div fxHide.gt-xs fxFlex="100" fxLayout="row" fxLayoutAlign="space-between center" class="app-toolbar">
         <div>
-          <mat-icon svgIcon="logo_icon" class="app-toolbar__logo"></mat-icon>
+          <img src="assets/icons/Logo.svg" class="app-toolbar__logo"/>
```
Comment (eddiejaoude) [inline web/src/app/app.component.html]: I think there should be a space before the close tag

```html
<img src="assets/icons/Logo.svg" class="app-toolbar__logo" />
```

File: web/src/app/main/components/help/help.component.html [hunk 3]
```diff
@@ -20,15 +20,15 @@
       <mat-divider></mat-divider>
       <mat-nav-list *ngFor="let help of filteredTopics">
         <a [routerLink]="['/help', help.path]">
-          <mat-icon class="help__card__content__icon" svgIcon="{{ help.icon }}"></mat-icon>
+          <mat-icon class="help__card__content__icon material-icons-outlined grey-color">{{help.icon}}</mat-icon>
```
Comment (eddiejaoude) [inline web/src/app/main/components/help/help.component.html]: Should be spaces around the `{{ help.icon }}`

File: web/src/app/app.component.html (hunk 1 above)
Comment (webkhushboo) [reply web/src/app/app.component.html]: done

File: web/src/app/app.component.html (hunk 2 above)
Comment (webkhushboo) [reply web/src/app/app.component.html]: done

File: web/src/app/app.component.html [hunk 4]
```diff
@@ -83,7 +83,7 @@
       </div>
       <div fxHide.gt-xs fxFlex="100" fxLayout="row" fxLayoutAlign="space-between center" class="app-toolbar">
         <div>
-          <img src="assets/icons/Logo.svg" class="app-toolbar__logo"/>
+          <img src="assets/icons/logo.svg" class="app-toolbar__logo" />
```
Comment (eddiejaoude) [inline web/src/app/app.component.html:86]: I think the `svg` file will need to be renamed too?

File: web/src/app/app.component.html [hunk 5]
```diff
@@ -163,13 +163,13 @@
...
             <div (click)="showDoorbell()" class="footer__button">
-              <mat-icon svgIcon="feedback_toolbar_icon"></mat-icon>
+              <mat-icon class="material-icons-outlined orange-color">feedback</mat-icon>
               Feedback
             </div>
           </div>
         </div>
         <div fxLayout="row" fxLayoutAlign="center center">
-          <mat-icon [style.width.px]="120" svgIcon="assets:dashboardhub"></mat-icon>
+          <img src="assets/icons/logo-horizontal-std.svg" [style.width.px]="120"/>
```
Comment (eddiejaoude) [inline web/src/app/app.component.html]: Please add a space before the closing of the tag in all places

```html
<img src="assets/icons/logo-horizontal-std.svg" [style.width.px]="120" />
```

File: web/src/assets/Logo.svg [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="198.138" height="31.324" viewBox="0 0 198.138 31.324">
```
Comment (eddiejaoude) [inline web/src/assets/Logo.svg]: This file will need renaming to `logo.svg`

File: web/src/app/app.component.html (hunk 4 above)
Comment (webkhushboo) [reply web/src/app/app.component.html:86]: I did but I forgot to push.

File: web/src/app/app.component.html (hunk 5 above)
Comment (webkhushboo) [reply web/src/app/app.component.html]: okay

File: web/src/assets/Logo.svg (hunk 6 above)
Comment (webkhushboo) [reply web/src/assets/Logo.svg]: I forgot to push ..

Comment (webkhushboo) [reply web/src/assets/Logo.svg]: Its not visible in the changes

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**